commit 7cd657f4d820de4454ed66bdab2c8bb43d56a33a
Author: Muellni <mmuellen@gmx.de>
Date:   Tue May 31 00:40:27 2011 +0200

    fix a lot of redefinition warnings if M_SQRT1_2 is already defined

Index: oce/inc/Standard_math.hxx
===================================================================
--- oce.orig/inc/Standard_math.hxx
+++ oce/inc/Standard_math.hxx
@@ -32,9 +32,12 @@
 #if defined(M_PI_2)
 # undef M_PI_2
 #endif
+# define M_PI_2     1.57079632679489661923
 
+#if defined(M_SQRT1_2)
+# undef M_SQRT1_2
+#endif
 # define M_SQRT1_2  7.0710678118654752440E-1
-# define M_PI_2     1.57079632679489661923
 
 # endif  /* __MATH_WNT_H */
 
